src/docs/funds.yaml

```diff
@@ -0,0 +1,137 @@
+tags:
```

Comment:
I think this is an old file from an old dev branch because we have only a `fund.yaml` file in the dev branch right now and it includes what is inside of this file. So, just make sure to delete this while merging your code with the dev branch.